bridge-balance-difference/bot-config.json.example

```diff
@@ -0,0 +1,16 @@
+{
+  "contract1": {
```

Comment:
what bridge is this monitoring?

Reply:
It's monitoring the anyswap bridge for USDC between ETH and AVAX